pb/paste/model.py

```diff
@@ -38,12 +38,9 @@ def _put(stream):
     b = stream.read()
     digest = sha1(b).hexdigest()
     size = len(b)
-    try:
-        if stream.getbuffer().nbytes > 2 ** 23:
-            b = get_fs().put(b)
```

Comment:
If the original line 43 is evaluated, the value of `b` is an ObjectId (indirection to a GridFS document). If it is not evaluated, the value of `b` is the content, without that indirection.

With your change applied, the behavior difference is that the content of **all** pastes are stored in gridfs, whereas the original code is intended to use gridfs on pastes larger than 8MiB.

Reply:
And, the reason for not using GridFS for all pastes boils down to "poorly justified optimization".

Reply:
I see now! I make a litle change to use [getvalue()](https://docs.python.org/2/library/io.html#io.BytesIO.getvalue) instead of `nbytes` so we still have the same functionality but using different approach.